src/zc/form/browser/tests.py

```diff
@@ -162,15 +197,21 @@ def pageSetUp(test):
     )
 
 
+optionflags = (
+    doctest.NORMALIZE_WHITESPACE
+    | doctest.ELLIPSIS
+    | doctest.REPORT_NDIFF
+    | doctest.IGNORE_EXCEPTION_DETAIL
+    | doctest.REPORT_ONLY_FIRST_FAILURE
+)
+
+
 def test_suite():
     return unittest.TestSuite([
         unittest.defaultTestLoader.loadTestsFromName(__name__),
-        doctest.DocFileSuite(
-            'exceptionviews.rst',
-            setUp=zope.component.testing.setUp,
-            tearDown=zope.component.testing.tearDown),
```

Comment:
I don't like this.  Even if the file is left only for documentation purposes, it would be nice to ensure it doesn't bit-rot.

If `u` prefixes cause you trouble, you can [wave a magic wand](https://github.com/zopefoundation/zope.viewlet/blob/43520bbd7fb375acaac86d0e2f351d193af757db/src/zope/viewlet/tests.py#L80-L86) and make them go away.

Reply:
I think the problem is not the `u` prefix – as this could also be solved using `print` – but the unicode char in one of the assertions.

The file is not actually tested documentation, so I'd suggest to remove it as you converted it to unittests.

Reply:
It is this unicode char problem. I tried printing re-encoding, comparing binaries. All seemed differently, so it was easier to convert them. It is truely not much documentation value in this file, so I will delete it.